packages/devtools_app/test/shared/table_test.dart

```diff
@@ -962,6 +962,52 @@ void main() {
       expect(find.byWidget(table), findsOneWidget);
     });
 
+    testWidgets('displays wide data with many columns',
+        (WidgetTester tester) async {
+      const strings = <String>[
+        'All work',
+        'and no play',
+        'makes Ben',
+        'a dull boy',
+        // ignore: no_adjacent_strings_in_list
+        'The quick brown fox jumped over the lazy dog, although the fox '
+        "couldn't jump very high and the dog was very, very small, so it really"
+        " wasn't much of an achievement on the fox's part, so I'm not sure why "
+        "we're even talking about it."
+      ];
```

Comment:
This has now become the best easter egg in our codebase.

Reply:
I'm glad you appreciate it! 🤣